pilot/PilotUserService.swift

```diff
@@ -36,14 +36,17 @@ class PilotUserService: NSObject {
                       completion: @escaping (PilotUser) -> Void,
                       failure: @escaping (HTTPStatusCode) -> Void) {
         
+        let basicCredentials = "\(PilotConfiguration.Thunder.userKey):\(PilotConfiguration.Thunder.userSecret)"
+            .data(using: String.Encoding.utf8)!.base64EncodedString(options: [])
+        
```

Comment:
It looks like you are still using it. Do you need it in the router then? Just want to make sure we're not doing the same thing twice.